extensions/amp-subscriptions/0.1/test/test-crypto-handler.js

```diff
@@ -116,19 +116,6 @@ describes.realWin(
       });
     });
 
-    describe('decryptDocumentContent_', () => {
```

Comment:
Hmm. I would kind of not mind this test :)

Reply:
Changed the test to call decryptAesGcm directly. This way, we have one test testing the function.